### PR TITLE
Add command to migrate republished content from meta to paragraphs

### PR DESCRIPTION
**Pull request by abdelmalekkkkk**
I created this command for this [ticket](https://app.asana.com/0/1201792624707312/1202664404571760). All it does is take the original's content link and name from the 'aalink' and 'aaname' custom fields, and add a paragraph to the end of each article with a link to that original article.

I also made the script `sleep` every 100 posts to avoid any issues with the database.

### CODE REVIEW

File: src/Migrator/PublisherSpecific/LkldNowMigrator.php [hunk 1]
```diff
@@ -162,6 +171,95 @@ public function cmd_lkldnow_migrate_avatars() {
...
+		);
+
+		$query = new WP_Query( $args );
+
+		$updated_posts_count = 0;
+
+		foreach ( $query->posts as $index => $post ) {
+			if ( ( $index  + 1 ) % 100 == 0 ) {
+				WP_CLI::log( 'Sleeping for 1 second... ');
+				sleep( 1 );
```
Comment (iuravic) [inline src/Migrator/PublisherSpecific/LkldNowMigrator.php:213]: Was the reason behind this sleep to minimize MySQL impact on Atomic? If so, thanks for thinking about this, and for being se thoughtful!

However I think you can feel free to skip this sleep here. We'll mostly want to sleeps when we do raw MySQL queries, and should split them with offsets & limits, too.

No "request changes" here are necessary -- if you're able to migrate the content with this sleep amount, feel free to continue. If you find it's taking too long, just comment it out.

File: src/Migrator/PublisherSpecific/LkldNowMigrator.php [hunk 2]
```diff
@@ -162,6 +171,95 @@ public function cmd_lkldnow_migrate_avatars() {
...
+		$log_filename = 'lkld_updated_articles.log';
+
+		$already_updated_meta_key = '_newspack_original_content_appended';
+
+		$source_link_meta_key = 'aalink';
+		$source_name_meta_key = 'aaname';
+
+		$paragraph_template = '
+		
+		<!-- wp:paragraph -->
```
Comment (iuravic) [inline src/Migrator/PublisherSpecific/LkldNowMigrator.php:188]: @abdelmalekkkkk , make sure you test these in real life, too.

From my experience, Gutenberg can be very sensitive to extremely simple things like extra spaces, like the tabs at the beginning of 		 ..." here. If it works, that's great, and no issues, just sharing my previous experience 👍

File: src/Migrator/PublisherSpecific/LkldNowMigrator.php [hunk 3]
```diff
@@ -162,6 +171,95 @@ public function cmd_lkldnow_migrate_avatars() {
...
+				esc_url( $source_link ),
+				esc_html( $source_name ),
+			);
+
+			$new_post_content = $post->post_content . $formatted_paragraph;	
+
+			$this->log( $log_filename, sprintf( "New content for post #%d:\n%s", $post->ID, $new_post_content ) );
+
+			if ( ! $dry_run ) {				
+				$result = wp_update_post( array(
```
Comment (iuravic) [inline src/Migrator/PublisherSpecific/LkldNowMigrator.php:245]: I was once told that $wpdb->update() is quite faster than this. If you'll like, do give it a try in some of your next migrators. P.S. $wpdb->update() can also use a wp_cache_flush() once it's done running.

**Review by iuravic — APPROVED** [review]
Looks great! I added some comments here too, but those are not "requested changes", just sharing my thoughts and advices :)

File: src/Migrator/PublisherSpecific/LkldNowMigrator.php (hunk 1 above)
Comment (abdelmalekkkkk) [reply src/Migrator/PublisherSpecific/LkldNowMigrator.php:213]: Correct. There weren't many posts so the sleep didn't cause much of an issue. Better late than sorry 😁

**Comment by abdelmalekkkkk** [comment]
Your comments are very helpful. Thanks!

**Pull request merged**